modules/openapi-generator/src/main/resources/cpp-pistache-server/model-header.mustache

```diff
@@ -51,6 +43,8 @@ public:
     {{/required}}
     {{/vars}}
 
+	friend void to_json(nlohmann::json& j, const {{classname}}& o);
```

Comment:
fix indent

Reply:
Replaced tabs with spaces in model-header.mustache and model-source.mustace. Regenerated samples.